ReactNativeClient/lib/QuickActions.js

```diff
@@ -0,0 +1,45 @@
+
+import {DeviceEventEmitter} from 'react-native';
+import QuickActions from 'react-native-quick-actions';
+const { _ } = require('lib/locale.js');
+
+export default (dispatch, folderId) => {
+	QuickActions.setShortcutItems([
+		{type: 'New note', title: _('New note'), icon: 'Compose'},
+		{type: 'New to-do', title: _('New to-do'), icon: 'Add'},
```

Comment:
Here it should be:

```
		{type: 'New note', title: _('New note'), icon: 'Compose', userInfo: { url: "" }},
		{type: 'New to-do', title: _('New to-do'), icon: 'Add', userInfo: { url: "" }},
```

Due to this bug: https://github.com/jordanbyron/react-native-quick-actions/issues/90

Also please add a command and link to that issue so that if we add more actions we don't forget to add the empty `userinfo` key.

Reply:
Sure thing, I added a comment with a link to the issue. I wasn't quite sure what you meant by "command" (maybe typo of "comment"?), so lmk if I missed something.